src/Modules/AuthenticationModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,7 +1,164 @@
-import React from 'react'
+import { Box, Button, Grid, TextField, Typography } from "@mui/material";
+import React from "react";
+import forgetImg from "../../../../assets/Images/forget-reset.png";
+import Alert from "@mui/material/Alert";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm, SubmitHandler } from "react-hook-form";
+import axios from "axios";
+import { useToast } from "../../../Context/ToastContext";
+import { getErrorMessage } from "../../../../utils/error";
 
+type Input = {
+  email: string;
+};
 export default function ForgetPass() {
+  const { showToast } = useToast();
+
+  const navigate = useNavigate();
+  let {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Input>();
+  const onSubmit: SubmitHandler<Input> = async (data) => {
+    try {
+      const response = await axios.post(
+        "https://upskilling-egypt.com:3000/api/v0/admin/users/forgot-password",
+        data
+      );
+      showToast("success", response.data.message);
+      navigate("/resetpass");
+
+    } catch (error: any) {
+      const err = getErrorMessage(error);
+      showToast("error", err);    }
+  };
   return (
-    <div>ForgetPass</div>
-  )
+    <>
+      <Grid
+        container
+        display="flex"
+        justifyContent="center"
+        
+      >
+        <Grid
+          item
+          xs={12}
+          md={6}
+          sm={12}
+          sx={{  ml: "50px"}}
+        >
+          <Typography sx={{ m: "15px", color: "#152C5B", fontWeight: 700 }}>
```

Comment:
We should use the spacing system in MUI as I mentioned above. The same goes for the colors. Using the color system will make it easier for us to update our colors and implement dark/light mode without any effort.